docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.8"
```

Comment:
Please describe what is the purpose of this docker file.

Reply:
I use it for testing or developing. In order not to add async api to the mocked server, I decided to add a real standalone one. For me, it looks cheaper from time and effort perspectives. It's almost compliant with Amazon S3. Also, tests will be run in a closer environment to production, so they become much trustworthy.

Moreover, you can add a step to github actions where minio can be run before the step with running the tests.

Reply:
The complexity is not worth it.

Adding another dependency for all developers is not a choice. Please mock all APIs instead of using docker.

I decide to run `only one command` to test the whole project before each commit fastly.